packages/wonder-blocks-modal/src/components/__docs__/modal-launcher.stories.js

```diff
@@ -0,0 +1,380 @@
+// @flow
+import * as React from "react";
+
+import Button from "@khanacademy/wonder-blocks-button";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {ActionMenu, ActionItem} from "@khanacademy/wonder-blocks-dropdown";
+import {LabeledTextField} from "@khanacademy/wonder-blocks-form";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import {ModalLauncher, OnePaneDialog} from "@khanacademy/wonder-blocks-modal";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {Body, Title} from "@khanacademy/wonder-blocks-typography";
+
+import type {StoryComponentType} from "@storybook/react";
+
+import type {ModalElement} from "../../util/types.js";
+import ModalLauncherArgTypes from "./modal-launcher.argtypes.js";
+
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+
+const customViewports = {
+    phone: {
+        name: "phone",
+        styles: {
+            width: "320px",
+            height: "568px",
+        },
+    },
+    tablet: {
+        name: "tablet",
+        styles: {
+            width: "640px",
+            height: "960px",
+        },
+    },
+    desktop: {
+        name: "desktop",
+        styles: {
+            width: "1024px",
+            height: "768px",
+        },
+    },
+};
+
+const DefaultModal = (): ModalElement => (
+    <OnePaneDialog
+        title="Single-line title"
+        content={
+            <View>
+                <Body>
+                    {`Lorem ipsum dolor sit amet, consectetur
+                        adipiscing elit, sed do eiusmod tempor incididunt
+                        ut labore et dolore magna aliqua. Ut enim ad minim
+                        veniam, quis nostrud exercitation ullamco laboris
+                        nisi ut aliquip ex ea commodo consequat. Duis aute
+                        irure dolor in reprehenderit in voluptate velit
+                        esse cillum dolore eu fugiat nulla pariatur.
+                        Excepteur sint occaecat cupidatat non proident,
+                        sunt in culpa qui officia deserunt mollit anim id
+                        est.`}
+                </Body>
+            </View>
+        }
+    />
+);
+
+export default {
+    title: "Modal/ModalLauncher",
+    component: ModalLauncher,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        viewport: {
+            viewports: customViewports,
+            defaultViewport: "desktop",
+        },
+        chromatic: {
+            viewports: [320, 640, 1024],
+        },
+    },
+    argTypes: ModalLauncherArgTypes,
+};
+
+export const Default: StoryComponentType = (args) => (
+    <ModalLauncher modal={DefaultModal}>
```

Comment:
**suggestion:** Spreading `...args` here would help us to try out the `Default` story with different combinations.

```suggestion
    <ModalLauncher modal={DefaultModal} {...args}>
```

Reply:
Oops, I actually meant to remove args altogether because I thought it doesn't need any of them specified. But I just tested it out and that would mean that none of the props would even work! Thanks for pointing this out.